ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_permissions_mgmt.robot

```diff
@@ -263,3 +268,18 @@ ${username} Should Not Have Access To ${project_title}
     Project Should Not Be Listed    project_title=${project_title}
     RoleBinding Should Not Exist    project_title=${project_title}
     ...    subject_name=${username}
+
+Assign ${permission_type} Permissions To User ${username} in Project ${project_title}
+    Open Data Science Projects Home Page
+    Wait Until Project Is Listed    project_title=${project_title}
```

Comment:
it's missing the `[Argument]   ${project_title}`

Reply:
It doesn't miss [Argument] section, as all arguments already embedded in the keyword name.
See also:
https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#embedding-arguments-into-keyword-name

Reply:
ops, sorry I misread it